pages/Hydra.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Hydra Ecosystem | Hydraecosystem.org
+permalink: /Hydra
+---
+
+# What is Hydra?
+
+As per the <a href="https://www.hydra-cg.com/spec/latest/core/">draft</a> proposed here, Hydra describes a lightweight vocabulary which aids in creation of hyper-media driven APIs. Vocabulary described by Hydra is based on linked data which helps machines to understand semantics behind the data they send or receive.
+
+# What's the big deal?
+
+Hydra is a set of vocabularies that can be used to document REST APIs so that users don't need to go through the additional API documentation (which is sometimes too complex) in order to understand working of the API. You might be wondering that “what’s so special about it?”, A direct answer is that it helps us document APIs such that they are readable by machines. One of the biggest problems faced by the programmers right now is that when they use APIs which are not readable by machines, they generally need to hardcode API structure/links on client side in order to display data we received via response from server. So in case you are using an API , Let's say a weather API in your application. Now suppose there is a change in API URL/Structure for some unknown reason (possible reasons can include scaling, etc). Examples shown below may make this even clearer. 
+
+```
+# API (Old)
+
+URL: www.wtrresults.in/Delhi
+
+{
+  temperature: "20C",
+  humidity: "10%",
+}
+
+# API (URL Updated) 
+
+URL: www.wtrresults.com/Delhi
```

Comment:
I don't thing the agent will be able to figure this change out since the domain name is different. The agent can only detect changes in API structure not the domain.